src/components/Set/Summary/index.tsx

```diff
@@ -0,0 +1,205 @@
+    </>
+  );
+};
+
+const SetAuthors = ({ authors }: { authors: Array<string> }) => {
+  if (!authors) return null;
+  return (
+    <tr>
+      <td>Authors</td>
+      <td data-testid="set-type">{authors.join(', ')}</td>
+    </tr>
+  );
+};
+
+type Props = {
+  data: {
+    metadata: SetMetadata;
+  };
+  loading: boolean;
+};
+
+const SummarySet = ({ data, loading }: Props) => {
+  const metadata: SetMetadata =
+    loading || !data.metadata
+      ? {
+          accession: '',
+          name: { name: '' },
+          description: '',
+          id: '',
+          source_database: '',
+          authors: null,
+          literature: [],
+          counters: {},
+          relationships: {
+            nodes: [],
+            links: [],
+          },
+        }
+      : data.metadata;
+  let currentSet = null;
+  if (metadata.source_database) {
+    for (const db of setDBs) {
+      if (db.name === metadata.source_database) currentSet = db;
+    }
+    if (metadata.source_database === 'panther')
+      metadata.description = metadata.name.name;
+  }
+
+  return (
+    <div className={css('vf-stack', 'vf-stack--400')}>
+      {metadata.relationships &&
+        metadata.relationships.nodes &&
+        metadata.relationships.nodes.map((m) => (
+          <SchemaOrgData
+            key={m.accession}
+            data={{ data: m, db: metadata.source_database }}
+            processData={schemaProcessData}
+          />
+        ))}
+      <section className={css('vf-grid', 'summary-grid')}>
+        <div className={css('vf-stack')}>
+          <table className={css('vf-table', 'left-headers')}>
+            <tbody>
+              <tr>
+                <td style={{ maxWidth: '50%' }}>Accession</td>
+                <td data-testid="set-accession">
+                  <Accession accession={metadata.accession} />
+                </td>
+              </tr>
+              <tr>
+                <td>Data type</td>
+                <td data-testid="set-type">Set</td>
+              </tr>
+              <tr>
+                <td style={{ width: '200px' }} data-testid="set-memberdb">
+                  Member database
+                </td>
+                <td>{currentSet?.dbName || metadata.source_database}</td>
+              </tr>
+              <SetAuthors authors={metadata.authors || []} />
+            </tbody>
+          </table>
+        </div>
+        {currentSet?.url_template ? (
+          <div className={css('vf-stack')}>
+            <section>
+              <h5>External Links</h5>
+              <ul className={css('no-bullet')} data-testid="set-external-links">
+                <li>
+                  <BaseLink
+                    id={metadata.accession}
+                    className={css('ext')}
+                    target="_blank"
+                    pattern={currentSet.url_template}
+                  >
+                    View {metadata.accession} in {currentSet.dbName}
```

Comment:
```suggestion
                    {currentSet.dbName}
```

Consistent with other external links.